n2y/rich_text.py

```diff
@@ -152,31 +158,6 @@ def to_pandoc(self):
             return annotated_ast
 
 
-def mock_notion_annotations(annotations=None):
```

Comment:
I had actually pulled these two mocks out since they're used in the core n2y code; now that you moved the other ones over, I moved it into the mocks file.